app/src/main/java/io/intrepid/contest/screens/contestcreation/categorieslist/CategoriesListPresenter.java

```diff
@@ -2,32 +2,40 @@
 
 import android.support.annotation.NonNull;
 
+import java.util.ArrayList;
 import java.util.List;
 
 import io.intrepid.contest.base.BasePresenter;
 import io.intrepid.contest.base.PresenterConfiguration;
 import io.intrepid.contest.models.Category;
 import io.intrepid.contest.models.Contest;
 
-
-class CategoriesListPresenter extends BasePresenter<CategoriesListContract.View> implements CategoriesListContract.Presenter {
-
+class CategoriesListPresenter extends BasePresenter<CategoriesListContract.View> implements CategoriesListContract.Presenter, CategoryClickListener {
     private final Contest.Builder contestBuilder;
+    private final String categoryName = "Score";
+    private final String categoryDescription = "Overall Submission Score";
 
     CategoriesListPresenter(@NonNull CategoriesListContract.View view,
                             @NonNull PresenterConfiguration configuration,
                             Contest.Builder contestBuilder) {
         super(view, configuration);
+        List<Category> categories = contestBuilder.categories;
+        if (categories == null || categories.isEmpty()) {
+            contestBuilder.categories = new ArrayList<>();
+            contestBuilder.categories.add(new Category(categoryName, categoryDescription));
+        }
         this.contestBuilder = contestBuilder;
     }
 
     @Override
+    public void onViewCreated() {
```

Comment:
You just want to make sure that `onViewCreated()` calls `displayCategories()`.
Also double check if `displayCategories()` is being called anywhere or if it can be private. (In which case the displayCategoriesTest should call `onViewCreated()` instead of `displayCategories()`.)